lib/routes/oauth.js

```diff
@@ -70,6 +70,102 @@ module.exports = (log, config, oauthdb) => {
         return oauthdb.getScopedKeyData(sessionToken, request.payload)
       }
     },
+    {
+      method: 'POST',
+      path: '/oauth/authorization',
+      options: {
+        auth: {
+          strategy: 'sessionToken'
+        },
+        validate: {
+          payload: {
+            client_id: validators.clientId.required(),
+            redirect_uri: Joi.string()
+              .max(256)
+              .uri({ scheme: ['https'] })
```

Comment:
We should allow http redirect_uri's for local development, right? Over in the oauth-server directory, there is a [`redirectUri` validator](https://github.com/mozilla/fxa-auth-server/blob/f6b303a40179a49c3c3ecc36ce56acdada50063e/fxa-oauth-server/lib/validators.js#L54), maybe it's a good time to move that into a shared location, though oddly, the [oauth-server's authoriation route](https://github.com/mozilla/fxa-auth-server/blob/f6b303a40179a49c3c3ecc36ce56acdada50063e/fxa-oauth-server/lib/validators.js#L54) doesn't use it, but the [token route](https://github.com/mozilla/fxa-auth-server/blob/f6b303a40179a49c3c3ecc36ce56acdada50063e/fxa-oauth-server/lib/routes/token.js#L62) does.